src/Routes/SamplePage/SamplePage.js

```diff
@@ -18,48 +14,87 @@ import './sample-page.scss';
  * https://reactjs.org/docs/components-and-props.html
  * https://medium.com/@thejasonfile/dumb-components-and-smart-components-e7b33a698d43
  */
-const SamplePage = () => {
 
-    const dispatch = useDispatch();
+class SamplePage extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { systems: [] };
+        this.getSystemsForRos = this.getSystemsForRos.bind(this);
+    }
 
-    const handleAlert = () => {
-        dispatch(
-            addNotification({
-                variant: 'success',
-                title: 'Notification title',
-                description: 'notification description'
-            })
+    getSystemsForRos() {
+        fetch(INVENTRY_API_ROOT.concat('/hosts'))
+        .then(res => res.json())
+        .then(
+            result => {
+                this.setState({
+                    systems: result.results
+                });
+            },
+            error => {
+                this.setState({
+                    error
```

Comment:
Currently, this error state is not displayed to the user. Can we use [alert compoment](https://www.patternfly.org/v4/components/alert/) to display the error.